docs/overview.rst

```diff
@@ -0,0 +1,56 @@
+PIE Service Overview
+--------------------
+The program intent and engagement (PIE) service is a data store used to capture
+various interactions a user could have with a program and its courses.
+
+Users of the service can query it for a user's intents, or create new intents
+via HTTP request or management command.
+
+Program Intent Model
+~~~~~~~~~~~~~~~~~~~~
+The program intent model is meant to represent when a learner marked an interest in a program, why we believe
+their action represents an intent, and how certain we are of their intent to participate in a program.
+
+The types of intent reasons and their corresponding certainty are described in the following table
+
++----------------------+--------------------------------------------+-------------+
+| Reason               | Description                                | Certainty   |
++======================+============================================+=============+
+| BUNDLE_PURCHASE      | User purchased program as a bundle         | CERTAIN_YES |
++----------------------+--------------------------------------------+-------------+
+| ENROLLED_ALL_COURSES | User has enrolled in all program courses   | MAYBE       |
++----------------------+--------------------------------------------+-------------+
+| EARNED_PROGRAM_CERT  | User has earned a program certificate      | CERTAIN_YES |
```

Comment:
@alangsto Maybe this isn't relevant to the user currently for the purpose of docs, but do we have any examples of CERTAIN_NO?

Reply:
We don't have any current examples of `CERTAIN_NO`, but I think it's probably a good thing to call out in the docs! I'll add a note